darts/models/forecasting_model.py

```diff
@@ -680,6 +695,10 @@ def predict(self,
 class ExtendedForecastingModel(ForecastingModel, ABC):
     """ The base class for "extended" forecasting models, handling optional exogenous variables.
 
+    Extended forecasting models expand upon the functionality of `ForecastingModel` in 2 ways:
+    1. They introduce an optional `exog` time series parameter which can be used as a covariate.
```

Comment:
The main difference with global models here is that `exog` denotes "future" covariates - i.e., covariates which are known for the next `horizon` time steps at prediction time. Whereas global models handle "historic" covariates (i.e., covariates that have the same time axes as the input series). We kept the name `exog` here (and not `covariates`) to try making it clearer that it's not quite the same thing; and also because statsmodels calls it `exog` (for the future covariates).

Reply:
I see. Do you think we should rethink the naming once we introduce future covariates in the torch models?

Reply:
Yes, if we can find a better naming I'm up for it 👍